module/x/gravitynft/types/msgs.go

```diff
@@ -15,9 +15,11 @@ var (
 	_ sdk.Msg = &MsgSendNFTToCosmosClaim{}
 	_ sdk.Msg = &MsgERC721DeployedClaim{}
 	_ sdk.Msg = &MsgExecuteIbcNFTAutoForwards{}
+	// TODO: There should probably only be one MsgSendNFTToEth..?
```

Comment:
One is needed to deposit nft in cosmos account and another one is for validators to register that nft has been unlocked/minted on eth side.